cmd/kube-apiserver/app/server.go

```diff
@@ -238,19 +238,39 @@ func (s *APIServer) Run(_ []string) error {
 		glog.Fatalf("Failure to start kubelet client: %v", err)
 	}
 
-	disableV1beta3 := false
-	v1beta3FlagValue, ok := s.RuntimeConfig["api/v1beta3"]
-	if ok && v1beta3FlagValue == "false" {
-		disableV1beta3 = true
+	// "api/all=false" allows users to selectively enable specific api versions.
+	disableAllAPIs := false
+	allAPIFlagValue, ok := s.RuntimeConfig["api/all"]
```

Comment:
Updated https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/cluster_management.md#turn-on-or-off-an-api-version-for-your-cluster and also updated the help message for runtime_config flag.